hypothesis_1.ipynb

```diff
@@ -11,12 +11,20 @@
     "In this notebook we aim to find answer to research questions by interacting with the data sources relevant to it, __the related statement is__:\n",
     "> Cultural context content account for a relevant portion of Wikipedia articles (7%-49%) and is less shared on average, which is causing the language imbalance on Wikipedia. Mass media-dominated content, which is more dominated by certain cultures, is more widely shared, and therefore, translated from larger and global languages more often.\n",
     "\n",
+    "AW:\n",
+    "* Should link to the Diversity Observatory to give foundational background for \"cultural context content\".\n",
+    "* It doesn't seem necessary to say that articles are \"shared\", there are many drivers for more popular content, for example search is a bigger driver than sharing.\n",
+    "* I'm not sure we can easily combine theories about CCC with pageviews and with flow imbalances in this way.  Some CCC attracts pageviews, some doesn't.  The suggested language pair is independent of pageviews.  The article suggester seems to be using pageviews as its main input weight, so any choice of language pair will result in popular articles being suggested, it seems irrelevant whether CCC is also present.\n",
+    "* Perhaps you can avoid stating such a complex hypothesis here, and focus on statments that are directly related to the questions answered in the notebok.\n",
```

Comment:
It makes sense to add statements related to what will be answered. I have addressed this in my new commit.